src/components/TimelineEvent.js

```diff
@@ -2,9 +2,14 @@ import React from 'react';
 import './TimelineEvent.css';
 import Timestamp from './Timestamp';
 
-const TimelineEvent = () => {
-  // Fill in your code here
-  return;
+const TimelineEvent = (props) => {
+  return (
+  <section>
+    <h3>{props.person}</h3>
```

Comment:
Nice use of headers here for styling!

Reply:
Question for you. Does your page show timestamps of a year ago or a few seconds ago? If it's a few seconds, try changing your timestamp to timeStamp (with a capital S).

Reply:
Awesome thanks!